base/setup/usetup/usetup.h

```diff
@@ -33,10 +33,11 @@
 
 /* PSDK/NDK */
 #define WIN32_NO_STATUS
+#define _KERNEL32_ // To define WINBASEAPI empty
 #include <windef.h>
 #include <winbase.h>
 #include <winreg.h>
-#include <winuser.h>
+#include <winuser.rh>
```

Comment:
is this a typo?

Reply:
No it's not (btw. if it were a typo, there would be a build error). The winuser.rh file contains only the defines for resources+ VK_* (which are used in usetup) and some other stuff, already present in winuser.h , but _**without**_ the user32 typedefs, enums, and function prototypes that are of no use in usetup.

Reply:
weird stuff.